### ISSUE TITLE
Can run ubuntu/debian/centos on ish, just like termux on android does?

### ISSUE THREAD

**Issue by bnuzhouwei** [ish-app/ish#1702]
Can run ubuntu/debian/centos on ish, just like termux on android does?

termux can install ubuntu/debian/centos/alpline on it, 
Can ish support them?

I want to install mysql run netcore apps etc.

**Comment by Rhinozz** [comment]
No. iSH uses Alpine for reasons documented at the bottom of the "[Install & Activate Alternate Filesystems](https://github.com/ish-app/ish/wiki/Install-&-Activate-Alternate-Filesystems) " wiki page:

> Most filesystems don't work yet because /bin/login or /sbin/init encounters a fatal error. In addition, some filesystems will never work in iSH because they don't have /sbin/init. These types of filesystems need to be patched before they can work in iSH.

**Comment by ClickNinYT** [comment]
I think you can, just you need a special version of iSH and know how to patch the Root File System of the distribution you want to run so it meets iSH's requirement then I think you can run it.

**Comment by atikattar1104** [comment]
Android User Here. Termux Uses Something Called `proot` And A Wrapper For The Same Called `proot-distro`. Check Them Out Here :- 

1) `https://github.com/termux/proot`

Wiki :- `https://wiki.termux.com/wiki/PRoot`

2) `https://github.com/termux/proot-distro`
